source/developer/dev-setup-centos-7.rst

```diff
@@ -0,0 +1,98 @@
+.. _dev-setup-centos-7:
+
+Installing Developer Components on CentOS 7
+===========================================
+
+Set up your development environment for building, running, and testing Mattermost.
+
+1. If you are developing with the Docker container, install and configure Docker CE:
+
+  a. Follow the Docker installation instructions at https://docs.docker.com/engine/installation/linux/docker-ce/centos/.
+
+  b. Edit your ``/etc/hosts`` file to include the following line:
+
+    ``127.0.0.1     dockerhost``
+
+  c. Add your username to the *docker* group so that you can run Docker without using sudo. Replace *{username}* with your username.
+
+    ``sudo gpasswd -a {username} docker``
+
+  d. Restart the Docker daemon.
+
+    ``sudo service docker restart``
+
+  e. Change your current group ID to the *docker* group.
+
+    ``newgrp docker``
+
+2. Install the development tools package and libpng12 required by pngquant(a library used by Mattermost).
+
+    a. ``sudo yum group install "Development Tools"``
+
+    b. ``sudo yum install -y libpng12``
+
+3. Download and install Go 1.8 for Linux:
+
+    a. Download the Go binary.
+
+       ``wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz``
```

Comment:
> On the other hand, the audience for this doc is developers, so if wget isn't on their system, they know what to do.

That was my initial though but I guess I should add wget along with libpng as I did, because even though the target is developers its not only experienced developers that use the dev version, some new comers and even less experienced people also use it to get start with something, and ended up being a good help to Mattermost so having it more complete is better.

Thanks for pointing it out, doing the change now.